components/aws/sagemaker/tests/integration_tests/component_tests/test_deploy_component.py

```diff
@@ -101,6 +105,20 @@ def test_create_endpoint(
             ]
             == "InService"
         )
+        if "ExpectedInstanceType" in test_params.keys():
+            new_endpoint_config_name = sagemaker_utils.describe_endpoint(sagemaker_client, input_endpoint_name)[
```

Comment:
[describe_endpoint](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.describe_endpoint) already returns production variants

Reply:
also add a comment saying what we intend to assert/verify

Reply:
describe_endpoint does not return the InstanceType variant, so we need describe_endpoint_config. Added the assert comment